index.js

```diff
@@ -19,14 +19,19 @@ const baseLink = "https://www.icloud.com/shortcuts/";
  * A shortcut metadata.
  */
 class ShortcutMetadata {
-	constructor(object) {
-		// Not sure if we keep the native Apple names for the properties?
-		this.WFWorkflowClientRelease = object.WFWorkflowClientRelease;
-		this.WFWorkflowMinimumClientVersion = object.WFWorkflowMinimumClientVersion;
-		this.WFWorkflowIcon = object.WFWorkflowIcon;
-		this.WFWorkflowImportQuestions = object.WFWorkflowImportQuestions;
-		this.WFWorkflowTypes = object.WFWorkflowTypes;
-		this.WFWorkflowInputContentItemClasses = object.WFWorkflowInputContentItemClasses;
+	constructor(metadata) {
+		this.client = {
+			minimumVersion: metadata.WFWorkflowMinimumClientVersion,
+			version: metadata.WFWorkflowClientRelease,
+		};
+		this.icon = {
+			startColor: metadata.WFWorkflowIcon.WFWorkflowIconStartColor,
+			imageData: metadata.WFWorkflowIcon.WFWorkflowIconImageData,
+			glyphNumber: metadata.WFWorkflowIcon.WFWorkflowIconGlyphNumber,
+		};
+		this.importQuestions = metadata.WFWorkflowImportQuestions;
```

Comment:
I haven't seen the structure for them myself, but can you make a `ImportQuestion` class, so we could have an array of those in here?